doc/examples/reconst_mapmri.py

```diff
@@ -0,0 +1,92 @@
+"""
+================================================================
+Continuous and analytical diffusion signal modelling with MAPMRI
+================================================================
+
+We show how to model the diffusion signal as a linear combination
+of continuous functions from the MAPMRI basis [Ozarslan2013]_.
+We also compute the analytical Orientation Distribution Function (ODF).
+
+First import the necessary modules:
+"""
+
+from dipy.reconst.mapmri import MapmriModel
+from dipy.viz import fvtk
+from dipy.data import fetch_isbi2013_2shell, read_isbi2013_2shell, get_sphere
```

Comment:
I would suggest to use a real dataset with this tutorial. It will attract more attention.